coalib/testing/LocalBearTestHelper.py

```diff
@@ -186,9 +204,10 @@ def check_results(self,
                                   tempfile_kwargs=tempfile_kwargs,
                                   settings=settings)
         if not check_order:
-            self.assertEqual(sorted(bear_output), sorted(results), msg=msg)
+            self.assertComparableObjectsEqual(
+                sorted(bear_output), sorted(results))
         else:
-            self.assertEqual(bear_output, results, msg=msg)
```

Comment:
It looks like you are no longer using `msg`, in which case you should remove the code above which creates `msg`.

But I think that is very misguided.  This method has a special case `if results in [[], ()]:`, which does not need to use "comparable" `Result`s because it is expected that there are no results, and thus any `Result` has nothing to be compared against.